web/app/themes/mitlib-parent/js/sticky/LICENSE.md

```diff
@@ -1,21 +0,0 @@
-Copyright 2013 Anthony Garand
```

Comment:
If these license files are alongside the javascript files they refer to I don't think we should remove them.

I understand the desire to move all the libs to a single directory, but that is outweighed by the license explicitly saying we need to include the license file in at least one of the two that are being removed here. Could we create subdirectories for each under `js/libs` so we can retain the license files for these?

Reply:
Yeah, that's a fair point. I can restructure this to make separate folders inside `js/libs/` - that makes sense to now, IMO